apps/web/src/screens/Dashboard/RemoveLayout/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { type FC } from 'react';
+import { Image, StyleSheet, TouchableOpacity } from 'react-native';
+import { modalActions, Text, View } from '@walless/gui';
+import { ModalConfigs } from '@walless/gui';
+import { ExtensionDocument } from '@walless/store';
+import { db } from 'utils/pouch';
+import { router } from 'utils/routing';
+
+import RemoveSymbol from './RemoveSymbol';
+
+const RemoveLayout: FC<{ config: ModalConfigs }> = ({ config }) => {
+	const { name, networkMeta, _id } = config.context as ExtensionDocument;
+	const removeLayout = async () => {
```

Comment:
would be nicer to have a Valtio action that handle this instead of component level method